server/s3_get_bucket_policy_action.cc

```diff
@@ -93,8 +93,9 @@ void S3GetBucketPolicyAction::send_response_to_s3_client() {
     request->send_response(error.get_http_status_code(), response_xml);
 
   } else {
-    request->send_response(S3HttpSuccess200,
-                           bucket_metadata->get_policy_as_json());
+    std::string& response_xml = bucket_metadata->get_policy_as_json();
```

Comment:
its better to take the return value in const reference

Reply:
Lets fix it as part of https://jts.seagate.com/browse/EOS-13756 now. Need retriggering of jenkins again for this fix.

Reply:
sure. I assume you have already updated the ticket so that we do not miss this.